internal/client/client.go

```diff
@@ -775,6 +775,10 @@ func (a *AuthApiClient) ListTasks() ybmclient.ApiListTasksRequest {
 	return a.ApiClient.TaskApi.ListTasks(a.ctx, a.AccountID)
 }
 
+func (a *AuthApiClient) ListRbacRoles() ybmclient.ApiListRbacRolesRequest {
+	return a.ApiClient.RoleApi.ListRbacRoles(a.ctx, a.AccountID).RoleTypes("ALL").Limit(100)
+}
```

Comment:
Hi
So we have a feature flag for custom roles that affects the behaviour of this API
If the feature flag is off and we call this API with the "ALL" role types filter- we get a "404: Requested API not found"
In such a case I want to catch this NFE and try to call this API with the "SYSTEM" filter to try to list only the system roles
Whats the best way to go about this? Thanks in advance :)

Reply:
Maybe catch the error here and retry the request? https://github.com/yugabyte/ybm-cli/pull/135/files#diff-18363742b4f67cb68058e1b926b87dfdbeb6287fc1cd5bbfbfb061461b0d6455R58-R61

Reply:
Please let me know if the implementation looks okay, thanks!